encryptor/queryDataEncryptor.go

```diff
@@ -326,8 +326,16 @@ func (encryptor *QueryDataEncryptor) encryptUpdateQuery(ctx context.Context, upd
 	if len(tables) == 0 {
 		return false, nil
 	}
+
 	qualifierMap := NewAliasToTableMapFromTables(tables)
 	firstTable := tables[0].TableName
+
+	// MySQL/MariaDB dont support returning after update statements
+	// Postgres doest but expect only one table in tables expression, so we can take the firstTable for returning matching
```

Comment:
actually, postgresql allows more than 1 table:
`An expression to be computed and returned by the UPDATE command after each row is updated. The expression can use any column names of the table named by table_name or table(s) listed in FROM. Write * to return all columns.`